### PR TITLE
Add more queue actions

### LINKED ISSUE #30: Support retrying all crashed tasks
Add a button for "Retry All" at the top of the table 
image

### LINKED ISSUE #31: Add an option to remove an item from the crashed queue
Currently, the crash queue shows a RED number of crashes in the dashboard. Sometimes there are crashes that are fixed or known, and the user wants to resolve it, but they have to live with this scary red number for a couple of days until the crahsed task would be removed from the queue.

I suggest adding an <kbd>X</kbd> button that will remove individual tasks. I also suggest adding a button to clean the crashed queue (see #30, it's similar)

### PR DESCRIPTION
**Pull request by nazywam**
Add a new button for canceling crashed tasks as well as buttons for retrying and cancelling all tasks from a selected queue.
Not really sure about the positioning of the "restart/cancel all tasks" buttons, but I couldn't come up with anything better

Edit: Also, not terribly sure about the endpoint names (should we be using api or not?)

Closes #30
Closes #31 

image

### CODE REVIEW

File: karton/dashboard/app.py [hunk 1]
```diff
@@ -33,6 +32,26 @@
 karton = KartonBase(identity="karton.dashboard")
 
 
+def restart_tasks(tasks: List[Task]) -> None:
+    identity = "karton.dashboard-retry"
+    producer = Producer(identity=identity)
+
+    for task in tasks:
+        forked = task.fork_task()
```
Comment (msm-code) [inline karton/dashboard/app.py:40]: probably unnecessary temporary variable? :thinking:

File: karton/dashboard/app.py [hunk 2]
```diff
@@ -202,17 +221,45 @@ def get_queues_api():
     )
 
 
+@app.route("/<queue_name>/restart", methods=["POST"])
+def restart_queue_tasks(queue_name):
+    state = KartonState(karton.backend)
+    queue = state.queues.get(queue_name)
+    if not queue:
```
Comment (msm-code) [inline karton/dashboard/app.py:228]: what do you think about a helper for this? I.e. some kind of `get_queue(state)` that either returns a queue or :abort:s with 404

**Review by msm-code — APPROVED** [review]

File: karton/dashboard/app.py (hunk 1 above)
Comment (nazywam) [reply karton/dashboard/app.py:40]: :wew:

**Comment by ITAYC0HEN** [comment]
Sorry for not following this thread, I recommend the buttons to be aligned to the right, so somewhere in the same imagined column of Actions

**Pull request merged**